app.rb

```diff
@@ -0,0 +1,92 @@
+require './person'
+require './book_class'
+require './rental_class'
+require './student'
+require './teacher'
+require './classroom_class'
+require './capitalize_decorator'
+
+class App
+  def initialize
+    puts 'School Library Application'
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def create_book
+    print 'Enter the book name :'
+    title = gets.chomp
+    print 'Enter the author name :'
+    author = gets.chomp
+
+    @books << Book.new(title, author)
+    puts 'Book Created Successfully'
+  end
+
+  def list_books
+    return 'No books added to the library' unless @books.any?
+
+    text = ''
+    @books.each.with_index(1) do |book, i|
+      text += "\n#{i}) Title: \"#{book.title}\" Author: #{book.author}"
+    end
+    text
+  end
+
+  def create_person
+    print 'Enter the person name :'
+    name = gets.chomp
+    print 'Enter the person age :'
+    age = gets.chomp
+
+    @people << Person.new(age, name)
+    puts 'Person Created Successfully'
+  end
+
+  def list_people
+    return 'No person added to the library' unless @people.any?
+
+    text = ''
+    @people.each.with_index(1) do |people, i|
+      text += "\n#{i}) Name: \"#{people.name}\" Age: #{people.age}"
+    end
+    text
```

Comment:
- [ ] After creating a `person` that means a person will have his/her personal ID, when checking list of persons you have outputs should be:
` puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"`

Reply:
done